### ISSUE TITLE
Discussion: Assign a11y reviewers within the team

### ISSUE THREAD

**Issue by nainar** [ampproject/wg-components#18]
We are the UI and a11y WG, however we have varying levels of proficiency in a11y. 

This means that even a PR reviewed by the UI and a11y WG might miss checking for a11y. 

I want to suggest assigning a111y reviewers on the WG (and adding folks from the wider AMP community who can review a11y) who must LGTM code that affects a11y.

**Comment by kristoferbaxter** [comment]
Are we planning to recommend places for people to ramp up on `a11y` knowledge?

**Comment by nainar** [comment]
Is the question for the Working Group of for the general AMP Project?

If latter: We should have a reference page from CONTRIBUTING.md

For the WG, we should be doing more active audits of the components with a11y experts to help upscale knowledge.

cc @sparhami who suggested this.
